teps/0091-trusted-resources.md

```diff
@@ -336,39 +262,28 @@ metadata:
   name: config-trusted-resources
   namespace: tekton-pipelines
 data:
-  onError: "stopAndFail"
+  skip-verification: false
   cosign-pubkey-path: "/etc/signing-secrets/cosign.pub"
   kms-pubkey-path: "gcpkms://projects/<project>/locations/<location>/keyRings/<keyring>/cryptoKeys/<key>"
 ```
-`onError: [ stopAndFail | continue ]`. (Optional, default `continue`)
-`stopAndFail` will cause the build to fail if image verification fails,  `continue` will allow the build to continue even
-if verification fails.
 
-`skip-verification`. (Optional, default `true`), if true then directly return nil, if false then do the verfication
+`skip-verification`. (Optional, default `true`), if true then directly return nil, if false then do the verification
```

Comment:
API clarity:
IMO, this option should be `verify` with a default value of `false`. I find it easier to reason about `verify=true` / `verify=false` than to deal with the double-negative of `skip-verification=false`.

Reply:
yeah sure!! I want to change it to what you suggested yesterday but somehow forgot it. 😂